storage/remote/client.go

```diff
@@ -56,6 +56,7 @@ type ClientConfig struct {
 	HTTPClientConfig config_util.HTTPClientConfig
 }
 
+// ReadClient allows to use SAMPLES method read against remote server.
```

Comment:
uses the SAMPLES

It doesn't permit it, it does it.

Reply:
;po